src/App.tsx

```diff
@@ -146,7 +148,7 @@ export default function App() {
 				<button id={"play-button"} onClick={togglePlaying}>
 					{playing ? <PauseIcon /> : <PlayIcon />}
 				</button>
-				<input id={"timeline-track"} type={"range"} min={"0"} max={"494"} value={"0"} />
+				<input id={"timeline-track"} type={"range"} min={"0"} max={"494"} onChange={(e) => setFrame(parseInt(e.target.value))} value={frame} />
```

Comment:
For this to work with files other than our sample data, the max attribute will need to be set to markerFileData.frames.length - 1